jdi-light-angular-tests/src/test/java/io/github/epam/TestsInit.java

```diff
@@ -0,0 +1,30 @@
+package io.github.epam;
+
+import static com.epam.jdi.light.driver.WebDriverUtils.killAllSeleniumDrivers;
+import static com.epam.jdi.light.elements.composite.WebPage.openUrl;
+import static com.epam.jdi.light.elements.init.PageFactory.initSite;
+import static com.epam.jdi.light.settings.JDISettings.DRIVER;
+import static com.epam.jdi.light.settings.WebSettings.logger;
+
+import io.github.com.StaticSite;
+import io.github.epam.testng.TestNGListener;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Listeners;
+
+@Listeners(TestNGListener.class)
+public class TestsInit {
+
+    @BeforeSuite(alwaysRun = true)
+    public static void setUp() {
+        killAllSeleniumDrivers();
+        initSite(StaticSite.class);
```

Comment:
use 
openSite(StaticSite.class); 
instead of 
initSite(StaticSite.class);
openUrl(DRIVER.domain);

Reply:
Thnks you, replaced